src/main/webapp/app/overview/course-overview.service.ts

```diff
@@ -204,4 +230,32 @@ export class CourseOverviewService {
     setSidebarCollapseState(storageId: string, isCollapsed: boolean) {
         localStorage.setItem('sidebar.collapseState.' + storageId, JSON.stringify(isCollapsed));
     }
+
+    /**
+     * Converts workingTime property to a formatted string to be displayed in sidebar-cards
+     */
+    convertWorkingTimeToString(workingTime: number): string {
+        let workingTimeString = '';
+
+        if (workingTime) {
+            const hours = Math.floor(workingTime / 3600);
+            const minutes = Math.floor((workingTime % 3600) / 60);
+            const seconds = workingTime % 60;
+            if (hours > 0) {
+                workingTimeString += `${hours}h`;
+                if (minutes > 0) {
+                    workingTimeString += ` ${minutes} min`;
+                }
+            } else if (minutes > 0) {
+                workingTimeString += `${minutes} min`;
+                if (seconds > 0) {
+                    workingTimeString += ` ${seconds} sec`;
+                }
+            } else {
+                workingTimeString += `${seconds} sec`;
+            }
```

Comment:
Is there a reason we cannot just simplify this as follows:
```suggestion
            if (hours > 0) {
                workingTimeString += `${hours}h`;
            }
            if (minutes > 0) {
                workingTimeString += `${minutes} min`;
            }   
            if (seconds> 0) {
                workingTimeString += `${seconds} sec`;
            }
```

Reply:
No, we can directly remove it. I am using artemisDate pipe in my follow up PR, so this function will removed in the follow up PR.